Description: Allow older factory
 Upstream requires 4.2.0, which isn't in Debian yet, due to a bug with
 factory 4.1.x + flint 2.7.x, but this doesn't seem to be an issue
 with flint 2.6.x, which is what is available in Debian.
Author: Doug Torrance <dtorrance@debian.org>
Bug: https://github.com/Macaulay2/M2/issues/1964
Last-Update: 2021-05-28

--- a/M2/configure.ac
+++ b/M2/configure.ac
@@ -1381,7 +1381,7 @@
 if test $BUILD_factory = no
 then
     AC_MSG_CHECKING([whether factory library is installed])
-    FACTORY_MINVERSION=4.2.0
+    FACTORY_MINVERSION=4.1.0
     if $PKG_CONFIG --exists "factory >= $FACTORY_MINVERSION"
     then
 	FACTORY_NAME=factory
